examples/quickrefl/core/editors/importdatawidgets/famousloader/importlogic.cpp

```diff
@@ -24,6 +25,23 @@ std::vector<std::string> DataImport::split(const std::string &s, char delim) {
   return elems;
 }
 
+//! Cleans the vector of strings provided by removing empty parts
+void DataImport::clean(std::vector<std::string>& input) 
+{
+    std::vector<std::string>::iterator i = input.begin();
+    while(i != input.end())
+    {
+        if(*i == "")
+        {
+            i = input.erase(i);
+        }
+        else
+        {
+            ++i;
+        }
+    }
+}
+
```

Comment:
Consider extracting this and similar methods into standalone functions and providing unit tests. Consider refactoring in the functional style
```
vec.erase(std::remove_if(vec.begin(), vec.end(), [](auto x){return x.empty();}), vec.end());
```